src/content/docs/es/tutorials/add-content-collections.mdx

```diff
@@ -0,0 +1,383 @@
+
+    ```json title="tsconfig.json" ins={5,6}
+    {
+      // Nota: No se necesitan cambios si se utiliza "astro/tsconfigs/strict" o "astro/tsconfigs/strictest"
+      "extends": "astro/tsconfigs/base",
+      "compilerOptions": {
+        "strictNullChecks": true,
+        "allowJs": true
+      }
+    }
+    ```
+
+### Crear una colección para tus publicaciones del blog.
+
+3. Crea una nueva **colección** (directorio) llamado `src/content/posts/`.
+
+4. Mueve todas las publicaciones de tu blog (archivos `.md`) de `src/pages/posts/` en esta nueva colección.
+
+5. Crea el archivo `src/content/config.ts` para [definir un esquema](/es/guides/content-collections/#definiendo-un-esquema-de-colección) para la colección `postsCollection`. Para el código que existe en este tutorial, agrega el siguiente contenido al archivo para definir todas las propiedades del frontmatter utilizadas en las entradas de blog:
+
+    ```ts title="src/content/config.ts"
+    // Importa las utilidades de `astro:content`
+    import { z, defineCollection } from "astro:content";
+    // Define un `type` y un `schema` para cada colección
+    const postsCollection = defineCollection({
+        type: 'content',
+        schema: z.object({
+          title: z.string(),
+          pubDate: z.date(),
+          description: z.string(),
+          author: z.string(),
+          image: z.object({
+            url: z.string(),
+            alt: z.string()
+          }),
+          tags: z.array(z.string())
+        })
+    });
+    // Exporta un solo objeto `collections` con las colecciones registradas
+    export const collections = {
+      posts: postsCollection,
+    };
+    ```
+
+### Generando las páginas de una colección
+
+6. Crea un archivo de página llamado `src/pages/posts/[...slug].astro`. Los archivos Markdown y MDX no se convertirán en páginas automáticamente usando el sistema de rutas basado en archivos cuando estos viven en una colección, por eso debes crear una página encargada de generar un blog individual para cada publicación.
+
+7. Agrega el siguiente código para [consultar tu colección](/es/guides/content-collections/#consultando-colecciones), para hacer que el slug y el contenido de cada publicación del blog estén disponibles para cada página que se genere:
+
+    ```astro title="src/pages/posts/[...slug].astro"
+    ---
+    import { getCollection } from 'astro:content';
+    import MarkdownPostLayout from '../../layouts/MarkdownPostLayout.astro';
+
+    export async function getStaticPaths() {
+      const blogEntries = await getCollection('posts');
+      return blogEntries.map(entry => ({
+        params: { slug: entry.slug }, props: { entry },
+      }));
+    }
+
+    const { entry } = Astro.props;
+    const { Content } = await entry.render();
+    ---
+    ```
+
+8. La etiqueta `<Content />` renderiza tu publicación con el `layout` para archivos Markdown. Esto te permite especificar una plantilla común para todas tus publicaciones.
```

Comment:
```suggestion
8. La etiqueta `<Content />` renderiza tu publicación con la plantilla para páginas Markdown. Esto te permite especificar una plantilla común para todas tus publicaciones.
```

Reply:
Hola @Waxer59 , siendo que estamos señalando la propiedad `layout` en el código de abajo, en mi opinión, vale la pena dejar el concepto `AS IS` al menos una vez, para hacer énfasis en lo que se debe enfocar el lector en el bloque de código de ejemplo.